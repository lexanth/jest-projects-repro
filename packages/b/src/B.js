import React from 'react'
import { C } from '@jest-repro/c'

export const B = () => (
  <div data-testid="B">
    <C />
  </div>
)
