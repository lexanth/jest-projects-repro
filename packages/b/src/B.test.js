import { render } from '@testing-library/react'
import { B } from './B'
import React from 'react'

describe('B', () => {
  it('should render', () => {
    const { container } = render(<B />)
    expect(container).toMatchSnapshot()
  })
})
