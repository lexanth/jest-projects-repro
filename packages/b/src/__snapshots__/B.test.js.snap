// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`B should render 1`] = `
<div>
  <div
    data-testid="B"
  >
    <div
      data-testid="C"
    />
  </div>
</div>
`;
